tests/eth2/beacon/test_helpers.py

```diff
@@ -1123,35 +1112,55 @@ def _correct_slashable_attestation_params(
         fork):
     valid_params = copy.deepcopy(params)
 
-    key = "custody_bit_0_indices"
-    (poc_0_indices, poc_0_signatures) = _get_indices_and_signatures(
-        num_validators,
-        messages[0],
-        privkeys,
-        fork,
-        slot_to_epoch(params["data"].slot, epoch_length),
-    )
-    valid_params[key] = poc_0_indices
-
-    key = "custody_bit_1_indices"
-    # NOTE: does not guarantee non-empty intersection
-    (poc_1_indices, poc_1_signatures) = _get_indices_and_signatures(
+    (validator_indices, signatures) = _get_indices_and_signatures(
         num_validators,
         messages[1],
         privkeys,
         fork,
         slot_to_epoch(params["data"].slot, epoch_length),
     )
-    valid_params[key] = poc_1_indices
 
-    signatures = poc_0_signatures + poc_1_signatures
+    valid_params["validator_indices"] = validator_indices
+    valid_params["custody_bitfield"] = get_empty_bitfield(len(validator_indices))
+
+    signatures = signatures
     aggregate_signature = bls.aggregate_signatures(signatures)
 
     valid_params["aggregate_signature"] = aggregate_signature
 
     return valid_params
 
 
+def _corrupt_custody_bitfield_not_empty(params):
+    validator_indices_length = len(params["validator_indices"])
+    corrupt_custody_bitfield = get_empty_bitfield(validator_indices_length)
+    corrupt_custody_bitfield = set_voted(corrupt_custody_bitfield, 0)
+    return assoc(params, "custody_bitfield", corrupt_custody_bitfield)
+
+
+def _corrupt_validator_indices(params):
+    corrupt_validator_indices = (
+        params["validator_indices"][1],
+        params["validator_indices"][0],
+    ) + tuple(params["validator_indices"][1:])
```

Comment:
This seems to result in an extra index, is this intended?
```suggestion
    ) + tuple(params["validator_indices"][2:])
```